facts/config.py

```diff
@@ -56,3 +57,7 @@
 # Initialize the content of the config parameters
 for key, value in cfg_defaults.items():
     config.set('common', key, str(value))
+
+windowsize_facts = datetime.timedelta(seconds=10)
+
+fact_attributes = ['serverId', 'CpuValue', 'MemValue', 'DataTime']
```

Comment:
mmm, I like it, imaginative solution that give you detail about the data that you are using.